unison-core/src/Unison/ABT.hs

```diff
@@ -214,14 +215,15 @@ extraMap p (Term fvs a sub) = Term fvs a (go p sub) where
     Tm x -> Tm (fmap (extraMap p) (p x))
 
 pattern Var' v <- Term _ _ (Var v)
-pattern Cycle' vs t <- Term _ _ (Cycle (AbsN' vs t))
--- pattern Abs' v body <- Term _ _ (Abs v body)
+pattern Cycle' vs t <- Term _ _ (Cycle (AbsN' vs (Tm' t)))
+pattern Abs'' v body <- Term _ _ (Abs v body)
```

Comment:
😅 

I'm a bit allergic to primes, is there a better name we can use for this please?
Maybe `UnwrapAbs` or something since it's a one-way pattern?

What does this do differently from `Abs'`, `AbsN'` and `AbsNA'`? Things are a bit messy as is, so let's try our best not to make it any worse 😆

Reply:
I think it's probably fine as-is, yeah it's not a great name, but neither is `UnwrapAbs` IMO. :)

Reply:
I came to chime in that a lot of the "prime" names are inherited/legacy, and that changing the name in just one package but not in other similar ones will also make things worse in a sense. It looks like it's probably `Abs'` that should get the name change, but not necessarily in this PR. 😬

Reply:
To clarify, I propose we punt on `Abs''` and circle back on the names for that and the rest in the future; any objections?

Reply:
What do you mean by punt on `Abs''`?

Reply:
Sorry, I meant as in: time-box coming up with a different name, since I think they all deserve better names in a different PR.